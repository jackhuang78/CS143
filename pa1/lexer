#!/bin/sh
java -classpath /afs/ir/class/cs143/lib/java-cup-11a.jar:/afs/ir/class/cs143/lib/jlex.jar:.:/usr/java/lib/rt.jar:`dirname $0` Lexer $*
