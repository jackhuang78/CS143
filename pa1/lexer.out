#name "exp.cl"
#2 TYPEID TEST_NUMBER_____________
#3 INT_CONST 23
#3 '.'
#3 INT_CONST 45
#4 INT_CONST 0033
#5 INT_CONST 1232434335456213133245346549023823498397120930786408237678612873691287631231234234
#6 INT_CONST 1
#6 OBJECTID e10
#7 INT_CONST 0000002
#8 INT_CONST 0
#8 '.'
#8 INT_CONST 0004
#11 TYPEID TEST_WHITE_SPACE_____________
#15 ERROR "\\"
#15 OBJECTID b
#16 ERROR "\\"
#16 OBJECTID t
#17 ERROR "\\"
#17 OBJECTID n
#18 ERROR "\\"
#18 OBJECTID f
#19 ERROR "\\"
#19 ERROR "\\"
#20 ERROR "\\"
#20 OBJECTID c
#21 ERROR "\\"
#21 OBJECTID d
#22 ERROR "\\"
#22 OBJECTID e
#23 ERROR "\\"
#24 ERROR "Unterminated string constant"
#24 ERROR "\\"
#25 ERROR "\\"
#25 OBJECTID b
#25 ERROR "\\"
#25 OBJECTID t
#25 ERROR "\\"
#25 OBJECTID n
#25 ERROR "\\"
#25 OBJECTID f
#25 ERROR "\\"
#25 ERROR "\\"
#25 ERROR "\\"
#25 OBJECTID c
#25 ERROR "\\"
#25 OBJECTID d
#25 ERROR "\\"
#25 OBJECTID e
#25 ERROR "\\"
#27 ERROR "Unterminated string constant"
#28 TYPEID TEST_TRUE_FALSE____________
#29 BOOL_CONST true
#30 TYPEID True
#31 BOOL_CONST true
#32 BOOL_CONST false
#33 BOOL_CONST false
#34 BOOL_CONST false
#35 TYPEID False
#38 TYPEID TEST_INDENTIFIER____________
#39 OBJECTID helloworld
#40 OBJECTID helloWorld
#41 OBJECTID hello_world
#42 OBJECTID hello
#42 '-'
#42 OBJECTID world
#43 TYPEID HelloWorld
#44 OBJECTID hELLOWORLD
#45 ERROR "_"
#45 OBJECTID helloworld
#46 '-'
#46 OBJECTID helloworld
#47 INT_CONST 1
#47 OBJECTID helloworld
#48 OBJECTID helloworld1
#49 INT_CONST 1
#49 TYPEID Helloworld
#50 TYPEID Helloworld1
#51 OBJECTID helloworld
#51 ERROR "!"
#52 INT_CONST 123
#52 OBJECTID h
#56 TYPEID TEST_STRING_CONSTRUCTION_________________
#57 ERROR "'"
#57 OBJECTID hello
#57 OBJECTID world
#57 ERROR "'"
#58 STR_CONST "hello world"
#59 STR_CONST "hello world"
#60 STR_CONST "hello \b world"
#61 STR_CONST "hello \t world"
#62 STR_CONST "hello \f world"
#63 STR_CONST "hello r world"
#64 STR_CONST "hello v world"
#65 STR_CONST "hello world"
#66 ERROR "Unterminated string constant"
#66 STR_CONST "hello world"
#66 STR_CONST ""
#67 STR_CONST ""
#67 OBJECTID hello
#67 OBJECTID world
#68 ERROR "Unterminated string constant"
#68 STR_CONST ""
#68 STR_CONST "hello world"
#69 STR_CONST "hello \n world"
#70 STR_CONST "hello 0 world"
#71 STR_CONST "hello \t world"
#72 STR_CONST "hello\tworld"
#73 STR_CONST "hello 'yes' world"
#74 STR_CONST "hello \"yes"
#74 OBJECTID world
#75 ERROR "Unterminated string constant"
#75 STR_CONST "hello \\n world"
#77 ERROR "Unterminated string constant"
#77 OBJECTID world
#78 ERROR "Unterminated string constant"
#79 STR_CONST "hello \nworld"
#80 STR_CONST "hello (* world *)"
#83 TYPEID TEST_COMMENT___________
#86 OBJECTID test
#86 OBJECTID comment
#100 OBJECTID unterminated
#100 ERROR "Unmatched *)"
#102 TYPEID TEST_LONG_STRINGS___________
#104 STR_CONST "xy6IzAeELTIc5CxeJW5UmeJMKZrWNOaXrZ8oHbArV8YfLgZUmrNlU4ZdKfAL89R9yzzS6iQ0SoC9zoHEzKOYcMUsRXigp08DCEdt0T1sf6Ub8haUHpU4n4plA53bxuMkX4IczRBcyu6gzha5sOvgEa2oGCKc9Ttcw0ZHkvFg9AHahZryrEox9iluEWSX5iQ5r1kCQefdUX2Lp3oSfFS3zuQp5MMbx0gA8ibxxri3VNBeLNL9EikMOP16lfGM2miA3n3newlexPoXnT7qoArXGkJJd7oBMeFJz39O7BXE6jkLogUL2I2BiLGoEC1ZwLszpUl1N8NPn2G54AFu9YxgRIuB4YLqow5ZcpMFGWyCf5ocYuENt3tNSEKaQk9JLKBPDOxMqL78s2PlNRpcW30wd9x8EaMt0W2qCiPKQJHtrQbFboGQUP0w85jwwodDicLHvhmvN1D9fDZjwrKx5pQPm8UHAr6SACYJIdkBNJflWfqaZX00u2WJ8n9umVQK0JW6JLFJm9NCJ1I97IgaNS1mMsSmTEWpPacMmqqtSXdquwQfQUJXCpbBVgutSGDTZKtkTHsnyQlLxUoZAjcntPfkWz2c84FnbVNJa19dTMo13mJfY8UlJA19ezMgIkgfrgqheY4MIUAPNF7UeAqcdg6AgBqN76vwjFVyhpggABub2WJE1GZIPxYuJKuHHLg7fOXY5sPcLB98WDEEFm0sGSIatBRdC5truzIlx8SXjwt03KSykMdvc6Ysf7x6xnB4OtSZkA3CD3PWfvXGpYYJmzPZ15D7iADg4pXb6lOLj0HvqWAvXCQ8mBadylM922U7xOiZEqf0uN9WrkAolMoqdC8OYcrvjN0o6DgpDDg3eBsR3lhOvbCtjaT918Y8PW5WWUCDDKiCbYW6RpJyaMA0qxyAomsePy4dhrNIEZRI4HOLg5WfeTjtHDUC7SD1Zjtnr1o9T7pMCqFRIgh1bxCCB6XQXL16usBX3ayZbt3WpPp7fHU34wr1mMwOMVj9zAumJ1Bx"
#106 ERROR "String constant too long"
#108 ERROR "String constant too long"
#112 TYPEID REMAINING_CASES________
#113 OBJECTID self
#114 TYPEID SELF_TYPE
