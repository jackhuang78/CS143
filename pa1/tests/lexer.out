#name "eof_string.cl"
#1 ERROR "EOF in string constant"
